src/main/java/com/minecolonies/coremod/client/render/RenderArcherBarbarianMob.java

```diff
@@ -0,0 +1,49 @@
+package com.minecolonies.coremod.client.render;
+
+import com.minecolonies.coremod.client.model.ModelBarbarian;
+import com.minecolonies.coremod.entity.ai.mobs.EntityArcherBarbarian;
+import net.minecraft.client.renderer.entity.Render;
+import net.minecraft.client.renderer.entity.RenderLiving;
+import net.minecraft.client.renderer.entity.RenderManager;
+import net.minecraft.client.renderer.entity.layers.LayerBipedArmor;
+import net.minecraft.client.renderer.entity.layers.LayerHeldItem;
+import net.minecraft.util.ResourceLocation;
+import net.minecraftforge.fml.client.registry.IRenderFactory;
+
+import javax.annotation.Nonnull;
+
+/**
+ * Created by Asher on 5/6/17.
```

Comment:
You can remove the "created by" and add a comment explaining what this class does